src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -32,6 +32,11 @@
 abstract class DoctrineType extends AbstractType
 {
     /**
+     * @var DoctrineChoiceLoader[]
+     */
+    private static $choiceLoaders = array();
```

Comment:
should it really be static ? Sharing the cache between the ORM and ODM types could be an issue